lib/msee.js

```diff
@@ -244,15 +249,25 @@ function processToken(options) {
                     content += processToken(options);
                 }
             }
+
+            var pad = options.listItemPad;
+            if (orderedList) {
+                var first = options.orderedListItemPadTemplate.replace('x', String(ordinal));
```

Comment:
[`wcstring.padding`](https://github.com/martinheidegger/wcstring#wcstringpaddingprocess-width-padding) converts a padding "input" into a standard padding. would it make sense to first run `options. orderedListItemPadTemplate` through padding and then work on the first left entry?
```
pad = wcstring.padding(options.orderedListItemPadTemplate)
pad.first = pad.first.replace('x', String(ordinal))
```

Maybe a "clone" method would be good on the padding objects returned by `wcstring.padding` (which is an extension of [`varsize-string.padding`](https://github.com/martinheidegger/varsize-string#vsstringpa) ?!